docs/index.bs

```diff
@@ -2905,7 +2909,6 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
       1. Else if |request| is a <a>subresource request</a>, then:
           1. If |client|'s <a>active service worker</a> is non-null, set |registration| to |client|'s <a>active service worker</a>'s <a>containing service worker registration</a>.
           1. Else, return null.
-      1. Let |activeWorker| be |registration|'s <a>active worker</a>.
```

Comment:
As |registration| is set in either the non-subresource request block or the subresource request block, moving this assignment up to somewhere seems not good. I think it'd be better to just use |registration|'s active worker instead of |activeWorker| in https://github.com/w3c/ServiceWorker/pull/1062/files#diff-27b79860afe28f01aed4f1f6228367faR2888.